pkg/persistence/disk_persistence_test.go

```diff
@@ -499,3 +499,55 @@ func TestDiskPersistence_AppendToArchive(t *testing.T) {
 
 	cleanup()
 }
+
+func TestDiskPersistence_Delete(t *testing.T) {
+	diskPersistence, _ := NewDiskHandle(dataDir)
+	path := fmt.Sprintf("%s/%s", pathToCurrent, dirName1)
+
+	bytesToTest := []byte{115, 111, 109, 101, 10}
+
+	diskPersistence.Save(bytesToTest, dirName1, fileName11)
+
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		if err != nil {
+			t.Fatalf("Dir [%+v] was supposed to be created", path)
+		}
+	}
```

Comment:
Shouldn't we instead check if the file was created?

Something like:
```
	pathToFile := fmt.Sprintf("%s/%s/%s", pathToCurrent, dirName1, fileName11)

	if _, err := os.Stat(pathToFile); os.IsNotExist(err) {
		t.Fatalf("file [%+v] was supposed to be created", pathToFile)
	}
```

Reply:
The tests were confusing, I updated them in 286069a18defe0155fc24fd7b047eff5cafe4a2f and 22a0f19484663ed9a39425df79aa9c09ad154798.